### ISSUE TITLE
Ensure Templates/Drafts Directories Exist

### ISSUE THREAD

**Issue by no-reply** [curationexperts/epigaea#234]
Template interactions fail on the production server. This is probably because the directory doesn't exist.

**Comment by mark-dce** [comment]
I updated Capistrano to create and share the temp directories across deployments in #236 but am still getting an error on the server:

```
I, [2017-09-13T18:50:05.266369 #6085]  INFO -- : [7fdbf45b-1bc5-4f73-9d58-3dfeaf7b5b0d] Started GET "/templates" for 86.30.211.127 at 2017-09-13 18:50:05 -0400
I, [2017-09-13T18:50:05.269098 #6085]  INFO -- : [7fdbf45b-1bc5-4f73-9d58-3dfeaf7b5b0d] Processing by Hyrax::TemplatesController#index as HTML
I, [2017-09-13T18:50:05.274791 #6085]  INFO -- : [7fdbf45b-1bc5-4f73-9d58-3dfeaf7b5b0d] Completed 500 Internal Server Error in 6ms (ActiveRecord: 0.0ms)
F, [2017-09-13T18:50:05.275314 #6085] FATAL -- : [7fdbf45b-1bc5-4f73-9d58-3dfeaf7b5b0d]   
F, [2017-09-13T18:50:05.275391 #6085] FATAL -- : [7fdbf45b-1bc5-4f73-9d58-3dfeaf7b5b0d] NameError (uninitialized constant Tufts::Template
Did you mean?  Tempfile):
F, [2017-09-13T18:50:05.275425 #6085] FATAL -- : [7fdbf45b-1bc5-4f73-9d58-3dfeaf7b5b0d]   
F, [2017-09-13T18:50:05.275449 #6085] FATAL -- : [7fdbf45b-1bc5-4f73-9d58-3dfeaf7b5b0d] app/controllers/hyrax/templates_controller.rb:4:in `index'
```

**Comment by no-reply** [comment]
This is closed by #266
